src/main/org/deidentifier/arx/metric/Metric.java

```diff
@@ -1492,6 +1492,16 @@ public String toString() {
     }
 
     /**
+     * Calculates the score.
```

Comment:
Please also create an extension of class MetricConfiguration (also consider seralizability issues with this class) that indicates whether or not a score function is provided, e.g. boolean isScoreFunctionSupported();